src/core/ext/filters/client_channel/lb_policy/priority/priority.cc

```diff
@@ -472,11 +472,11 @@ void PriorityLb::ChoosePriorityLocked() {
   }
   // Did not find any child in CONNECTING, delegate to last child.
   SetCurrentPriorityLocked(
-      static_cast<int32_t>(config_->priorities().size() - 1),
+      static_cast<uint32_t>(config_->priorities().size() - 1),
```

Comment:
This cannot be `size_t` as it appears intended that priority is a unsigned 32 bit integer and `UINT32_MAX` is used in several placeless.

Reply:
@markdroth Hi Mark a friendly reminder to take a look at this PR as you provided previous feedbacks? Thanks!

Reply:
@yashykt Hey Yash! Would you be able to help with the review as well (Or recommend someone more appropriate)? Thanks!